modules/examples/seed/server/modules/protocol_avro/src/main/scala/PeopleService.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example.seed.server.protocol.avro
+
+import higherkindness.mu.rpc.protocol._
+
+@service(AvroWithSchema) trait PeopleService[F[_]] {
+
+  def getPerson(request: example.seed.server.protocol.avro.PeopleRequest): F[
```

Comment:
Same for the service, it should be generated by https://github.com/higherkindness/mu/pull/598/files#diff-cb75815fe8b5491e1c4b014d79883574R5

Reply:
right too